### ISSUE TITLE
Allow the reference date to be the value of another picker

### ISSUE THREAD

**Issue by ppazos** [stefangabos/Zebra_Datepicker#190]
I'm playing with pair and direction options and it doesn't seem quite right to define a generic datetime range:

1. picker A top limited by the value of picker B, if there is no value, do not limit
2. picker B bottom limited by the value of picker A, if there is no value, do not limit

It seems the `direction` option considers the reference date to today, so I can't set 1st day last month as the picker B value and let picker A be top limited by that value.

Not sure how this could be accomplished. Reading the docs the range pickers seem to be a missing feature "Date ranges (sort of)" https://stefangabos.github.io/Zebra_Datepicker/

To be able to use this picker, ranges is a requirement for my application. The library seems really good for my other use cases, I really like, for instance, the format being specified in the PHP way, and has many other useful options, just lacking on the range thing.

Thanks!

**Comment by stefangabos** [comment]
yes, setting the `pair` attribute doesn't work when set to both date pickers. that's why is eternally called `sort of` :)
might give this a try - proper pairing with two open picker - this year. i can commit to something more exact :)
but thanks for the feedback!
